dlrover/python/elastic_agent/torch/training.py

```diff
@@ -332,10 +337,15 @@ def _join_rendezvous(self):
         """The node join a rendezvous by sending its
         ID and local world size.
         """
-        round = self._client.join_rendezvous(
+        round, stage = self._client.join_rendezvous(
```

Comment:
It is not recommended to obtain job stage through the RPC method used for training rdzv. It would be more reasonable to acquire job stage information via the heartbeat mechanism.